docs/syntax/classDiagram.md

```diff
@@ -768,7 +768,28 @@ Beginner's tip—a full example using interactive links in an HTML page:
 
 ### Styling a node
 
-It is possible to apply specific styles such as a thicker border or a different background color to individual nodes. This is done by predefining classes in css styles that can be applied from the graph definition using the `cssClass` statement or the `:::` short hand.
+It is possible to apply specific styles such as a thicker border or a different background color to a node.
```

Comment:
Please modify the actual source doc linked at the top of this file, this is the auto generated one. 

Also, need the [available since version](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/community/code.md#3-update-documentation) tag.